.gitignore

```diff
@@ -125,4 +125,5 @@ app.*.symbols
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
-!/dev/ci/**/Gemfile.lock
\ No newline at end of file
+!/dev/ci/**/Gemfile.lock
+/integration_test/test_bundle.dart
```

Comment:
@silverkate Please remove it

Reply:
when we run patrol tests test_bundle.dart is generated automatically and it's recommended not to commit to version control

<img width="690" alt="image" src="https://github.com/VSydorko/flutter_guidelines/assets/55805761/b0b25efd-afcc-457f-9b66-339a17cd495a">